### DISCUSSION TITLE
Anyone use wz_mini_hacks for Home Assistant's Camera Integration?

### DISCUSSION THREAD

**Discussion by mkanet** [gtxaspec/wz_mini_hacks#652] [Q&A] [answered]
**The whole reason I want to get the Wyze V3 PAN is to add it to Home Assistant.**  I want to connect it to 

I'm curious if installing wz_mini_hacks and setting up z Wyze 3 Pan is as easy as it appears in the instructions.  Everything looks straightforward; however, what scared me were all the open issues here.

Are there any tips you can suggest before I purchase and install everything?

**Comment by vic4news** [comment, marked as answer]
I don’t own a pan v3, but I do have several v3 cams. I added them to home assistant using the generic camera integration. No issues

**Reply by tismofied** [reply]
I am sorry @vic4news for the tag from an old thread but I am failing at adding my camera to HA as generic camera.
would you mind sharing how you set it up ?

**Comment by bjs-pdx** [comment]
> **The whole reason I want to get the Wyze V3 PAN is to add it to Home Assistant.** I want to connect it to
> 
> I'm curious if installing wz_mini_hacks and setting up z Wyze 3 Pan is as easy as it appears in the instructions. Everything looks straightforward; however, what scared me were all the open issues here.
> 
> Are there any tips you can suggest before I purchase and install everything?

I don't think the Pan V3 works with mini_hacks

**Reply by bjs-pdx** [reply]
Supported hardware:

![image](https://github.com/user-attachments/assets/f4690e25-2f75-4e0e-bd5c-40c0ab0107d7)
